SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -639,6 +639,15 @@ void USpatialNetDriver::NotifyActorDestroyed(AActor* ThisActor, bool IsSeamlessT
 			// Remove it from any dormancy lists
 			ClientConnection->DormantReplicatorMap.Remove(ThisActor);
 		}
+
+		// Check if this is a dormant entity, and if so retire the entity
+		Worker_EntityId EntityId = PackageMap->GetEntityIdFromObject(ThisActor);
+		if (IsDormantEntity(EntityId) && ThisActor->HasAuthority())
+		{
+			// Deliberately don't unregister the dormant entity, but let it get cleaned up in the entity remove op process
+			check(StaticComponentView->GetAuthority(EntityId, SpatialGDK::Position::ComponentId) == WORKER_AUTHORITY_AUTHORITATIVE);
```

Comment:
This will cause a crash if the last update we sent before making the actor dormant caused it to cross a boundary.